ansible_base/lib/abstract_models/common.py

```diff
@@ -43,7 +43,7 @@ class Meta:
 
     modified = models.DateTimeField(
         editable=False,
-        help_text=_("The date/time this resource was created"),
+        help_text=_("The date/time this resource was modified"),
```

Comment:
Won't this cause `makemigrations` to make new migrations for a ton of things in test_app and other apps?

Reply:
AFAIK when using lazy translation django make migration will not be able to detect that there is a change on that text https://github.com/ansible/django-ansible-base/blob/92cc5e39f0dbf7b9dbc22381e433d387288a9e13/ansible_base/lib/abstract_models/common.py#L8 

I will double check, if it makes then I revert to the incorrect text and add a comment.

Reply:
I removed the fix, at some point someone else will probably end up fixing that when a migration is required.